js/tc39/tc39_test.go

```diff
@@ -163,6 +162,11 @@ var (
 		"test/language/expressions/compound-assignment/S11.13.2_A7.10_T2.js": true,
 		"test/language/expressions/compound-assignment/S11.13.2_A7.10_T1.js": true,
 		"test/language/expressions/assignment/S11.13.1_A7_T3.js":             true,
+
+		// timezone
```

Comment:
These seem to be disabled in goja (see https://github.com/dop251/goja/blob/master/tc39_test.go#L83C3-L86) since 2021, so I'm not sure why they started to fail now, and/or if this is the way to proceed.

cc/ @mstoykov

Reply:
how are they failing? I can't see them in the logs